### PR TITLE
Polish Validator Community doc

### PR DESCRIPTION
**Pull request by DrW3RK**
Just rephrasing the contents in the doc with Grammarly. Please check if the doc is readable and there is no ambiguity added.

### CODE REVIEW

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
One strategy for helping gain additional visibility is to set up a dedicated site for your validator, which
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
on the site. After setting up a website, a validator should add this website to the
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
to their validator stash. Slashing applies to the total stake of a Validator, therefore having a high self-stake shows confidence in the operations. This helps show commitment from the Validator as they have skin in the game and can be penalized for negative actions or poor maintenance.
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
these efforts so that the values of a Nominator and Validator are aligned.
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
policy for the software on that OS. For example, are LTS versions used? Do they use NiXOS,
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
#### Automation and orchestration approaches (Terraform, Ansible, Chef, Puppet, Kubernetes, etc.)
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
done? Outlining a desired network topology for a Validators infrastructure design will help Nominators understand how resilient their operations are to attacks. Some things to highlight are the usage of firewalls,
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
binaries. How do they get alerted for new releases? Do they receive updates from the matrix chatrooms? Do
```

File: docs/maintain/maintain-guides-validator-community.md
Comment (alfarok) [inline docs/maintain/maintain-guides-validator-community.md]:
```suggestion
care.  If compromised, both the validator and nominator can be slashed. Outlining how keys are
````

**Review by alfarok — COMMENTED** [review]
@DrW3RK Just added a few general suggestion, feel free to commit as you see fit.  I would also run prettier as a final step before merging as I noticed a few lengthy lines likely from inline edits.

**Comment by DrW3RK** [comment]
Thank you for the suggestions! The article reads much better now. Approve and merge if everything looks good

**Review by alfarok — APPROVED** [review]

**Pull request merged**
